app/users/resources.py

```diff
@@ -0,0 +1,95 @@
+import json
+
+from flask import request
+from flask_restful import Resource
+
+from app import db, User
+from app.lib.response_util import buildOkResponse
+
+
+def returnUser(user):
+    """Private method to convert a user model into a dictionary.
+
+        Args:
+        user: A user model.
+    """
+
+    return {
+        "id": user.id, "name": user.name
+    }
+
+
+class UserListResource(Resource):
+    """Class for creating and accessging users."""
+    def get(self):
+        """
+            This method gets a list of all the users.
+
+            Returns:
+            Method returns the a list of user names and ids in dictionary
+            format.
+        """
```

Comment:
If you would like, you could combine the method description and returns section.
```
"""Returns a list of all the users in an OK response.
"""
```

If you would like to keep them separate, please indent the returns section once and push the method description up one line:
```
"""This method gets a list of all the users.

Returns:
    Method returns the a list of user names and ids in dictionary
    format.
"""
```